apps/playwright/tests/admin/pool-candidate.spec.ts

```diff
@@ -94,6 +96,208 @@ test.describe("Pool candidates", () => {
+  });
+
+  test("Verification and notes mutations", async ({ appPage }) => {
+    await loginBySub(appPage.page, "admin@test.com");
+    await appPage.page.goto(`/en/admin/candidates/${candidate.id}/application`);
+    await appPage.waitForGraphqlResponse("PoolCandidateSnapshot");
+
+    // priority verification
+    await appPage.page
+      .getByRole("button", { name: "Edit Priority status" })
+      .click();
+    await appPage.page
+      .getByRole("radio", { name: "This claim has been verified" })
+      .click();
+    await appPage.page.getByRole("spinbutton", { name: "Year" }).fill("2030");
+    await appPage.page
+      .getByRole("combobox", { name: "Month" })
+      .selectOption("01");
+    await appPage.page.getByRole("spinbutton", { name: "Day" }).fill("25");
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await expect(
+      appPage.page.getByText(
+        /This claim has been verified, expires on January 25, 2030/i,
+      ),
+    ).toBeVisible();
+
+    // veteran verification
+    await appPage.page
+      .getByRole("button", { name: "Edit Veteran status" })
+      .click();
+    await appPage.page
+      .getByRole("radio", { name: "This claim does not apply to" })
+      .click();
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await expect(
+      appPage.page.getByText(/This claim does not apply to/i),
+    ).toBeVisible();
+
+    // notes
+    await appPage.page.getByRole("button", { name: "Add notes" }).click();
+    await appPage.page
+      .getByRole("textbox", { name: "Notes" })
+      .fill("Notes notes notes");
+    await appPage.page.getByRole("button", { name: "Save changes" }).click();
+    await appPage.waitForGraphqlResponse("PoolCandidate_UpdateNotes");
+    await expect(
+      appPage.page.getByRole("button", { name: "Edit notes" }),
+    ).toBeVisible();
+    await expect(appPage.page.getByText(/Notes notes notes/i)).toBeVisible();
+  });
+
+  test("Completing an assessment step", async ({ appPage }) => {
+    await loginBySub(appPage.page, "admin@test.com");
+    await appPage.page.goto(`/en/admin/candidates/${candidate.id}/application`);
+    await appPage.waitForGraphqlResponse("PoolCandidateSnapshot");
+
+    // to assess icon by application screening
+    await expect(
+      appPage.page.getByLabel("To assess").locator("path"),
+    ).toBeVisible();
+
+    // education result
+    await appPage.page
+      .getByRole("row", { name: "Education requirement To" })
+      .getByRole("button")
+      .click();
+    await expect(
+      appPage.page.getByText("I meet the applied work"),
+    ).toBeVisible();
+    await expect(
+      appPage.page
+        .getByLabel("Assess the candidate's")
+        .getByText("Test Experience"),
+    ).toBeVisible();
+    await appPage.page.getByText("Demonstrated", { exact: true }).click();
+    await appPage.page
+      .getByText("Work experience equivalency is accepted", { exact: true })
```

Comment:
I cannot seem to get the radio by name to work here. Not sure why
This isn't a regular radio input, but from the CardItem/CardGroup component

![image](https://github.com/user-attachments/assets/51c85295-12d6-4d01-a01b-1919061ebb85)

Reply:
oof, that is super unfortunate and kind of raises a red flag. Looks like these tests were a good idea! I don't think this is an actual issue and probably just a downside of playwright but I'll do some extra testing of these components after this to confirm.

Reply:
Aye aye

Reply:
I think the `CardOption`s in a group all share the same `name`